src/annotator/highlight-clusters.tsx

```diff
@@ -113,6 +134,83 @@ export class HighlightClusterController implements Destroyable {
     this._activate(this._isActive());
   }
 
+  /**
+   * Update nesting data- attributes on <hypothesis-highlight> and associated
+   * SVG highlight rects.
+   */
+  _updateClusterData() {
+    if (!this._isActive()) {
+      return;
+    }
+
+    /**
+     * Recursively walk a tree of <hypothesis-highlight> elements, adding
+     * nesting data attributes to both <hypothesis-highlight> and their
+     * associated SVG highlight (rect) elements. This allows styling based on
+     * nesting relationships and depth.
+     *
+     * @param highlightEls - A collection of sibling <hypothesis-highlight>
+     * elements
+     * @param parentCluster - The cluster value of the parent highlight to
+     * `highlightEls`, if any
+     * @param nestingLevel - The nesting "level", relative to the outermost
+     * <hypothesis-highlight> element (0-based)
+     * @param parentClusterLevel - The parent's nesting depth, per its cluster
+     * value (`parentCluster`). i.e. How many levels since the cluster value
+     * changed? This allows for nested styling of highlights of the same cluster
+     * value.
+     */
+    const _updateHighlightClusters = (
```

Comment:
```suggestion
    const updateHighlightClusters = (
```

Only private methods need a `_` prefix.

Reply:
Ah, yes, this is an artifact of moving this function, which was previously a private method.